samples/cellular/nrf_cloud_rest_fota/README.rst

```diff
@@ -111,7 +111,7 @@ To enable full modem FOTA, add the following parameter to your build command:
 ``-DOVERLAY_CONFIG=overlay_full_modem_fota.conf``
 
 Also, specify your development kit version by appending it to the board name.
```

Comment:
```suggestion
Also, if you are using an nRF9160 DK, specify your development kit version by appending it to the board name.
```

at this time i don't think we need to add the hw version for 9161 builds?
so you may consider these 2 suggestions.